elephant/phase_analysis.py

```diff
@@ -307,3 +307,80 @@ def phase_difference(alpha, beta):
     delta = alpha - beta
     phase_diff = np.arctan2(np.sin(delta), np.cos(delta))
     return phase_diff
+
+
+def weighted_phase_lag_index(signal_i, signal_j, sampling_frequency,
+                             absolute_value=True):
+    r"""
+    Calculates the Weigthed Phase-Lag Index (WPLI)
+
+    This function expects the two signals (each containing multiple trials).
+    It Fourier-transforms the signals and for each trial pair, it calculates
+    the cross-spectrum. With that the WPLI is calculated.
+
```

Comment:
This reads more like an algortihmic instruction on how to calculate the WPLI -- as a user, you would expect a more useful explanation of this measure. I think one should rephrase.